dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/util/SystemConfig.java

```diff
@@ -44,6 +44,7 @@
      * A JVM option for where files are stored on the file system.
      */
     public static final String FILES_DIRECTORY = "dataverse.files.directory";
+    public static final String TRUE = "true";
```

Comment:
Wouldn't something like `Boolean.valueof(String)` or `Boolean.parseBoolean(String)` be preferable?

Reply:
I think we should use `SettingsServiceBean.isTrueForKey` method here.